bin/generate_practice_exercise.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# shellcheck source=/dev/null
+source ./bin/generator-utils/utils.sh
+source ./bin/generator-utils/prompts.sh
+source ./bin/generator-utils/templates.sh
+
+# Exit if anything fails.
+set -euo pipefail
+
+# If argument not provided, print usage and exit
+if [ $# -ne 1 ]; then
+    echo "Usage: bin/generate_practice_exercise.sh <exercise-slug>"
+    exit 1
+fi
+
+# Check if sed is gnu-sed
+if ! sed --version | grep -q "GNU sed"; then
+    echo "GNU sed is required. Please install it and make sure it's in your PATH."
+    exit 1
+fi
+
+# Check if jq and curl are installed
+command -v jq >/dev/null 2>&1 || {
+    echo >&2 "jq is required but not installed. Aborting."
+    exit 1
+}
+command -v curl >/dev/null 2>&1 || {
+    echo >&2 "curl is required but not installed. Aborting."
+    exit 1
+}
+
+# Check if exercise exists in configlet info
+check_exercise_existence "$1"
+
+# ==================================================
+
+SLUG="$1"
+UNDERSCORED_SLUG=$(dash_to_underscore "$SLUG")
+EXERCISE_DIR="exercises/practice/${SLUG}"
+AUTHOR_NAME=$(get_author_name)
+message "info" "You entered: $AUTHOR_NAME. You can edit this later in .meta/config.json.authors"
+EXERCISE_NAME=$(get_exercise_name "$SLUG")
```

Comment:
I think it's fine to not ask people about this but just assume that the generated name is correct. You could then print the generated name and still mention that this could be changed in the config.json file.